app/config/config.yml

```diff
@@ -81,11 +84,12 @@ assetic:
                 - "%kernel.root_dir%/../vendor/owenversteeg/min/less/tables.less"
                 - "%kernel.root_dir%/../vendor/owenversteeg/min/less/messages.less"
             output: css/min.css
-
         common_css:
             inputs:
-              - "@WebsiteBundle/Resources/public/css/common.css"
+                - "bundles/website/css/common.css"
```

Comment:
@vkhramtsov I know it's little bit ugly, but in other way cssrewrite works not correct.